targets/ar71xx-generic

```diff
@@ -1,8 +1,6 @@
-ATH10K_PACKAGES=
-ATH10K_PACKAGES_QCA9887=
```

Comment:
I would like these variables always to be set, even when GLUON_WLAN_MESH has an invalid value. We could move the definitions from the "if 11s" to the top and only keep the "if ibss".